massa-network/src/peer_info_database.rs

```diff
@@ -17,22 +15,41 @@ use tokio::task::JoinHandle;
 use tokio::time::{sleep, Duration};
 use tracing::{trace, warn};
 
+/// Peer categories.
+/// There is a defined number af slots for each category.
+#[derive(Clone, Copy, Serialize, Deserialize, Debug, PartialEq, Eq)]
+pub enum PeerType {
+    /// if the peer is in bootstrap servers list
+    /// for now it is decoupled from the real bootstrap sever list, it's just parsed
+    /// TODO: https://github.com/massalabs/massa/issues/2320
+    Bootstrap,
+    /// Connection from these nodes are always accepted
+    WhiteListed,
+    /// Just a peer :pear:
+    Standard,
+}
+
+impl Default for PeerType {
+    fn default() -> Self {
+        PeerType::Standard
+    }
+}
+
 /// All information concerning a peer is here
 #[derive(Clone, Copy, Serialize, Deserialize, Debug)]
 pub struct PeerInfo {
     /// Peer ip address.
     pub ip: IpAddr,
-    /// If peer is banned.
-    pub banned: bool,
-    /// If peer is boostrap, ie peer was in initial peer file
-    pub bootstrap: bool,
+    /// The category the peer is in affects how it's treated.
+    pub peer_type: PeerType,
```

Comment:
in the future, this could be made a String, maybe as a followup

Reply:
see #2334